x-pack/elastic-agent/docs/elastic-agent-command-line.asciidoc

```diff
@@ -196,30 +310,34 @@ elastic-agent run [global-flags]
 
 These flags are valid whenever you run `elastic-agent` on the command line.
 
-`-c <string>`::
+`--c <string>`::
```

Comment:
`--c`?

Reply:
Works according to the command help (just trying to be consistent, but maybe sticking with -c makes more sense for existing users?) I will revert.

-c, --c string                     Configuration file, relative to path.config (default "elastic-agent.yml")